spec/values/calculation/atan2.hrx

```diff
@@ -0,0 +1,204 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `atan2()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> unitless/input.scss
+a {b: atan2(1, -10)}
+
+<===> unitless/output.css
+a {
+  b: 174.2894068625deg;
+}
+
+<===>
+================================================================================
+<===> compatible_units/input.scss
+a {b: atan2(1cm, -10mm)}
+
+<===> compatible_units/output.css
+a {
+  b: 135deg;
+}
```

Comment:
Also test:

* a real unit and a percentage
* a complex unit and a percentage (this should fail)
* two percentages
* two fake units
* a real unit and a fake unit

As a style consideration, I'd probably call these directories `units/none`, `units/compatible`, `units/real_and_unknown`, `units/complex_and_unknown`, `units/unknown`, `units/fake`, `units/real_and_fake`.